src/vmm/src/arch/x86_64/mod.rs

```diff
@@ -130,9 +131,11 @@ pub fn configure_system(
 
     match boot_prot {
         BootProtocol::PvhBoot => {
+            info!("Configuring VM for PVH boot.");
```

Comment:
Oh ok, you have the logging here. I would rather have it in `vmm::builder`, where you use the `LinuxLoader` results to figure out what kind of protocol to use.

You have also implemented `Display` on `BootProtocol`, so it would be enough to do something like:

```
debug!("Loading kernel using {boot_prot}");
```

please keep it a `debug!` rather than `info!`.

Reply:
Also, I don't think this change needs a separate commit, as done in this version.

Reply:
Ok, I've put `debug!("Kernel loaded using {boot_prot}");` into `vmm::builder::load_kernel`.  (Past tense because we don't know which protocol LinuxLoader used until after the kernel was loaded.)